src/pages/components/code-block.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Code block
+description: >
+  Code block is used to display text with code formatting. It uses syntax
+  highlighting to make code more legible.
+package: '@zendeskgarden/react-typography'
+components:
+  - typography/src/elements/CodeBlock.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To display read-only text with code formatting</li>
+      <li>To display code snippets for users to copy</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>For editing code. Code snippets are read-only.</li>
+      <li>
+        <Markdown>
+          For simple, single-line usage, use the [Code](/components/code) component
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+Code block renders code with syntax highlighting.
+
+import Default from '../../examples/typography/code-block/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/typography/code-block/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Light mode
+
+By default, Code block is dark. Shed some light on it using `isLight`.
+
+import Light from '../../examples/typography/code-block/Light.tsx';
+import LightCode from '!!raw-loader!../../examples/typography/code-block/Light.tsx';
+
+<CodeExample code={LightCode}>
+  <Light />
+</CodeExample>
+
+### Language
+
+Use the `language` prop to determine the syntax highlighting language. Default is `tsx`.
```

Comment:
"to determine" here sounds like the language prop performs some evaluation to learn what language is displayed. 

Perhaps:

> Use the `language` proper to specify the syntax highlighting language.